src/Stats/components/GameInfo.tsx

```diff
@@ -0,0 +1,17 @@
+import { useEffect, useState } from "react"
+import { IGameStatProps, RusGameNames } from "../../interfaces/interfaces"
+
+export const GameInfo:React.FC<IGameStatProps> = (props) => {
```

Comment:
You should pass object "props.stats.optional[props.name]" as props for this component.
`GameInfo:React.FC<IGameStatProps> = ({stats, name}) => {
const {correctAnswers, wrongAnswers, attempts, newWords, bestSeries} = stats;
}`